format/spec.md

```diff
@@ -1133,6 +1142,7 @@ Hash results are not dependent on decimal scale, which is part of the type, not
 4. UUIDs are encoded using big endian. The test UUID for the example above is: `f79c3e09-677c-4bbd-a479-3f349cb785e7`. This UUID encoded as a byte array is:
 `F7 9C 3E 09 67 7C 4B BD A4 79 3F 34 9C B7 85 E7`
 5. `doubleToLongBits` must give the IEEE 754 compliant bit representation of the double value. All `NaN` bit patterns must be canonicalized to `0x7ff8000000000000L`. Negative zero (`-0.0`) must be canonicalized to positive zero (`0.0`). Float hash values are the result of hashing the float cast to double to ensure that schema evolution does not change hash values if float types are promoted.
+6. `variant` values are currently not valid for bucketing and so they are not hashed.
```

Comment:
I don't think Notes is a good place for this. Can you start a new paragraph about unhashed types?

> A 32-bit hash is not defined for `variant` because there are multiple representations for equivalent values.